core/state/snapshot/snapshot.go

```diff
@@ -863,7 +865,7 @@ func (t *Tree) generating() (bool, error) {
 	return layer.genMarker != nil, nil
 }
 
-// DiskRoot is a external helper function to return the disk layer root.
+// DiskRoot is an external helper function to return the disk layer root.
```

Comment:
The impact of out-of-scope changes 😄

Reply:
Counterargument: [Boy Scout Rule](https://97-things-every-x-should-know.gitbooks.io/97-things-every-programmer-should-know/content/en/thing_08/) by Uncle Bob